parsers/mcmaster-parser-2.js

```diff
@@ -18,6 +18,68 @@ function convertTo24Hour(time) {
     return time.replace(/(AM|PM)/, '');
 }
 
+
+/**
+ * Takes any date and turns it into the correct format, so javascript gets the right date.
+ * // MM/DD/YYYY is the format the javascript object reads or YYYY/MM/DD
```

Comment:
why is there a a // here? it's already inside a function comment, just put them on the same line